examples/hashes/connector.py

```diff
@@ -0,0 +1,115 @@
+# This example shows how to calculate hash of fields to be used as primary key. This is useful in scenarios where the
+# incoming rows do not have any field suitable to be used as a Primary Key.
+# Fivetran recommends to define primary keys (https://fivetran.com/docs/connectors/connector-sdk/best-practices#declaringprimarykeys) when writing a connector.
+# This is to avoid disruptions caused due to a schema change (addition/removal of a column, etc) that might be needed in the maintainance phase of your connector.
+# See the Technical Reference documentation (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details.
+
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Operations as op
+
+# Import built-in Python modules
+import hashlib
+import json
+
+# Define the schema function which lets you configure the schema your connector delivers.
+# See the technical reference documentation for more details on the schema function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#schema
+# The schema function takes one parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+# In this example, there is no use of the configuration
+def schema(configuration: dict):
+    return [
+        {
+            "table": "user",  # Name of the table in the destination.
+            "primary_key": ["_fivetran_id"],  # Primary key column(s) for the table.
```

Comment:
let's not use Fivetran id, customer should make their own column and not our system column